tooling/api/src/main/java/org/jboss/windup/tooling/data/Classification.java

```diff
@@ -39,6 +39,12 @@
     Iterable<Link> getLinks();
 
     /**
+     * This contains a list of {@link Quickfix}s for follow up in tools
+     * @return
+     */
+    Iterable<Quickfix> getQuickfixes();
```

Comment:
Oh, I had forgotten that this was in the planning document. I'm not entirely sure what the use case would be, but I guess there is no harm in leaving it here. Maybe that would be suitable for file level replacements (XSLT, etc).